src/SignInComponent.tsx

```diff
@@ -0,0 +1,342 @@
+import React, { useState } from 'react';
+import { css } from "@emotion/core";
+import { body, headline } from '@guardian/src-foundations/typography/cjs';
+import { palette } from "@guardian/src-foundations";
+import { Button } from "@guardian/src-button";
+import { from, until } from '@guardian/src-foundations/mq';
+import { space } from '@guardian/src-foundations';
+import { SvgCross } from '@guardian/src-icons';
+import { SvgInfo } from '@guardian/src-icons';
+import { AppStore } from "./assets/app-store"
+import { PlayStore } from "./assets/play-store"
+
+const imgHeight = "300";
+
+export type Props = {
+  message: string;
+  onButtonClick: (buttonType: string) => void;
+  firstName?: string;
+};
+
+export const wrapper = css`
+  html {
+    box-sizing: border-box;
+  }
+  *,
+  *:before,
+  *:after {
+    box-sizing: inherit;
+  }
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  background-color: ${palette.background.secondary};
+  color: ${"#333"};
+`;
+
+export const contentContainer = css`
+    align-items: stretch;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    margin: 0 auto;
+    width: 100%;
+    max-width: 980px;
+    ${from.tablet} {
+        flex-direction: row;
+    }
+    ${from.desktop} {
+        min-height: ${imgHeight}px;
+    }
+    ${from.leftCol} {
+        max-width: 1140px;
+    }
+    ${from.wide} {
+        max-width: 1300px;
+    }
+`;
+
+export const topLeftComponent = css`
+    width: 93%;
+    padding: ${space[4]}px;
+    ${from.tablet} {
+        width: 55%;
+        padding-right: 0;
+    }
+    ${from.desktop} {
+        width: 60%;
+    }
+    ${from.wide} {
+        width: 60%;
```

Comment:
Is this needed given that we have the same rule in `from.wide` above?

Reply:
No - it's gone now.